platform/qemu.go

```diff
@@ -545,7 +545,8 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 		panic("Ignition specified but no primary disk")
 	}
 
-	if builder.Swtpm {
+	// TPM devices aren't on s390x/ppc64le at least
+	if builder.Swtpm && builder.Board == "amd64-usr" {
```

Comment:
This looks fine. I am not sure if TPM is supported on ARM. But for ppc64le and s390x looks like it will not be supported anytime soon.

Reply:
Based on... https://www.kernel.org/doc/html/latest/security/tpm/tpm_ftpm_tee.html maybe?  But I think the swtpm we use is probably emulating an Intel one?  Not sure.

Reply:
Anyways let's not block on this right now but I'll ask about it.

Reply:
agreed.

Reply:
I think software TPM2 should work just fine on ARM, if it doesn't let me know and I'll get it looked into. There's been a lot of work done with TPM2 on ARM so I suspect it should just work like on x86.

Reply:
https://github.com/coreos/mantle/pull/1158